packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -163,6 +163,24 @@ class ServiceConnectionManager {
     return listenable;
   }
 
+  String? _cachedRootLibKey;
+  String? _cachedRootLibValue;
+
+  /// Returns root library or empty string.
```

Comment:
looks like it returns root library or null if it cannot detect it. Update comment to match